src/gui/guiFormSpecMenu.cpp

```diff
@@ -529,6 +542,63 @@ void GUIFormSpecMenu::parseScrollBar(parserData* data, const std::string &elemen
 	errorstream<< "Invalid scrollbar element(" << parts.size() << "): '" << element << "'"  << std::endl;
 }
 
+void GUIFormSpecMenu::parseScrollBarOptions(parserData* data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if (parts.size() > 0) {
+		for (unsigned int i = 0; i < parts.size(); i++) {
+
+			std::vector<std::string> options = split(parts[i], '=');
+
+			if (options.size() != 2) {
+				warningstream << "Invalid scrollbaroptions option syntax: '" <<
+					element << "'" << std::endl;
+				continue; // Go to next option
+			}
+
+			int value = stoi(options[1]);
+
+			if (options[0] == "max") {
+				data->scrollBarOptions.max = value;
+				continue;
+			}
+			if (options[0] == "min") {
+				data->scrollBarOptions.min = value;
+				continue;
+			}
+			if (options[0] == "smallstep") {
+				if (value < 0)
+					data->scrollBarOptions.smallStep = 10;
+				else
+					data->scrollBarOptions.smallStep = value;
+				continue;
+			}
+			if (options[0] == "largestep") {
+				if (value < 0)
+					data->scrollBarOptions.largeStep = 100;
+				else
+					data->scrollBarOptions.largeStep = value;
+				continue;
+			}
+			if (options[0] == "thumbsize") {
+				if (value < 0)
```

Comment:
```suggestion
				if (value <= 0)
```
Else you would divide by 0.